docs/csharp/tutorials/intro-to-csharp/branches-and-loops.yml

```diff
@@ -1,11 +1,10 @@
 ### YamlMime:Tutorial
 title: Branches and loops
 metadata:
-  title:  Branches and loops interactive tutorial - C# quickstarts
-  description: In this quickstart about branches and loops, you'll use your browser to learn C# interactively. You'll write C# code and see the results of compiling and running your code directly in the browser.
+  title:  Branches and loops - Introduction to C# interactive tutorial
```

Comment:
In one place "interactive" was replaced with "online". What about changing it everywhere: "interactive tutorial" -> "online tutorial"?

Reply:
per above comment, the titles are different than the H1s